### PR TITLE
National youth council

### PR DESCRIPTION
**Pull request by Yush-Pokharel**
## What kind of change does this PR introduce? 
- [ ✔] Added a new website
 Added new website of National Youth Council and solved merge conflicts.

### CODE REVIEW

**Review by drishyakarki — APPROVED** [review]
LGTM

File: data/websites.json [hunk 1]
```diff
@@ -1060,6 +1060,11 @@
       "url": "https://nitc.gov.np/",
       "description": "Official Website of Nepal Government's NITC"
     },
+    {
+          "name": "National Youth Council",
```
Comment (susanhsrestha) [inline data/websites.json:1064]: Kindly use prettier to properly format the code and put the information in alphabetical order.

**Review by susanhsrestha — CHANGES_REQUESTED** [review]

File: data/websites.json (hunk 1 above)
Comment (Yush-Pokharel) [reply data/websites.json]: Corrected the alphabetical order. LGTM now.

**Review by susanhsrestha — CHANGES_REQUESTED** [review]
This commit contains multiple data duplicates. Review those, please, and submit the updated versions.

**Review by thenishantsapkota — APPROVED** [review]
LGTM!

**Review by drishyakarki — APPROVED** [review]

**Pull request merged**